src/equations/polytropic_euler_2d.jl

```diff
@@ -301,6 +301,48 @@ end
     return abs(v1) + c, abs(v2) + c
 end
 
+# Calculate maximum wave speed for local Lax-Friedrichs-type dissipation as the
+# maximum velocity magnitude plus the maximum speed of sound
+@inline function max_abs_speed_naive(u_ll, u_rr, orientation::Integer,
+                                     equations::PolytropicEulerEquations2D)
+    rho_ll, v1_ll, v2_ll = cons2prim(u_ll, equations)
+    rho_rr, v1_rr, v2_rr = cons2prim(u_rr, equations)
+
+    # Get the velocity value in the appropriate direction
+    if orientation == 1
+        v_ll = v1_ll
+        v_rr = v1_rr
+    else # orientation == 2
+        v_ll = v2_ll
+        v_rr = v2_rr
+    end
+    # Calculate sound speeds
+    c_ll = sqrt(equations.gamma * equations.gamma * rho_ll^(equations.gamma - 1))
+    c_rr = sqrt(equations.gamma * equations.gamma * rho_rr^(equations.gamma - 1))
```

Comment:
```suggestion
    c_ll = sqrt(equations.gamma * equations.kappa * rho_ll^(equations.gamma - 1))
    c_rr = sqrt(equations.gamma * equations.kappa * rho_rr^(equations.gamma - 1))
```
Because c^2 = gamma * p / rho and p = kappa * rho^gamma for the polytropic equations I believe one of these constants should be an `equations.kappa`.

Reply:
Correct!